submissions/f3d0t/task_js_dom/script.js

```diff
@@ -72,10 +69,10 @@ document.addEventListener("DOMContentLoaded", () => {
 		if (target.nodeName === "LI" && oldId !== currentId) {
 			nav.querySelector(".active")?.classList.toggle("active");
 			target.classList.toggle("active");
-			let newPokemon = getPokemonContent(currentId);
+			const newPokemon = pokemons.find((item) => item.id == currentId);
```

Comment:
why it calls `newPokemon` ? 
`pokemon` or `currentPokemon` is more meaningful

Reply:
Because , I think that currentPocemon - it's a pokemon that is currently on the screen. This variable holds whole new pokemon from list, and that's why i called it so. Just pokemon is also good I think